libraries/Microsoft.Bot.Builder/Teams/TeamsInfo.cs

```diff
@@ -246,10 +246,11 @@ public static Task<TeamsChannelAccount> GetMemberAsync(ITurnContext turnContext,
                 Activity = (Activity)activity,
             };
 
-            await ((BotFrameworkAdapter)turnContext.Adapter).CreateConversationAsync(
+            await turnContext.Adapter.CreateConversationAsync(
```

Comment:
TODO: review this one with @EricDahlvang

Reply:
We should probably change the signature of this method, so it uses AppCredentials instead of MicrosoftAppCredentials.

Reply:
I fixed the unit test that broke from my change to use cloud adapter, and the test was able to pass by successfully calling `CreateConversationAsync` on `CloudAdapterBase`.

@EricDahlvang, could you please verify the unit test change I made, and confirm if my test code is in line with the intended usage of this TeamsInfo method? I updated the method signature to accept `AppCredentials`.